src/metrics/site_level.jl

```diff
@@ -34,6 +34,7 @@ using Statistics
 
 rs = ADRIA.load_results("some results")
 tac = ADRIA.metrics.total_absolute_cover(rs)
+```
```

Comment:
Ending backticks are in the wrong place.

Next few lines are also example code.

Reply:
Oh, my bad. Ready for another review now.